app/models/miq_worker/runner.rb

```diff
@@ -275,17 +275,15 @@ def message_restarted(*_args)
     # just consume the restarted message
   end
 
-  def message_sync_active_roles(*args)
-    _log.info("#{log_prefix} Synchronizing active roles...")
-    opts = args.extract_options!
-    sync_active_roles(opts[:roles])
-    _log.info("#{log_prefix} Synchronizing active roles complete...")
-  end
-
   def message_sync_config(*_args)
     _log.info("#{log_prefix} Synchronizing configuration...")
     sync_config
     _log.info("#{log_prefix} Synchronizing configuration complete...")
+
+    _log.info("#{log_prefix} Synchronizing active roles...")
+    opts = _args.extract_options!
+    sync_active_roles(opts[:roles])
+    _log.info("#{log_prefix} Synchronizing active roles complete...")
```

Comment:
As discussed, let's just move this into the sync_config method itself (or perhaps let's just merge the two methods into message_sync_config and get rid of sync_config itself.